### ISSUE TITLE
Don't issue warning on nested scope blocks

### ISSUE THREAD

**Issue by wimrijnders** [google/styleguide#359]
Sorry to share a niggle of mine:

A nested block, with the intention of limiting the scope of a variable e.g. for mutex locking:

```c++
...
	{
		profiler_lock plck(m_profiler, "QueueClient.put_base.put");
		id = client.put(json, priority, delay, DEFAULT_TTR);
	}
...
```

...yields the warning: **Missing space before {  [whitespace/braces] [5]** on the opening brace.

First, I note that there is plenty of space before `{`, so the intention of this check must be elsewhere.

Second, I consider this a perfectly valid usage of blocks within `C++` and would appreciate it if the error doesn't pop up here.

I note that I would prefer not to resort to inline directives to prevent this, or having to disable the check because it _can_ be useful at times.

At the hope of not asking the impossible, can this particular case be fixed? TIA.

**Comment by AndrewMut** [comment]
@wimrijnders hey, where did u dissapear? sorry for disturbing here)) heh

**Comment by wimrijnders** [comment]
@AndrewMut Oh doing some things, this and that, bought me a house. Like, lurking in the real world from my administrator tasks :smile:.

**Comment by vapier** [comment]
public cpplint development in this repository has been halted (see https://github.com/google/styleguide/pull/837). please migrate to https://github.com/cpplint/cpplint now.
